pkg/proxy/definition_test.go

```diff
@@ -118,6 +122,26 @@ func testAddMiddlewares(t *testing.T) {
 	assert.Len(t, routerDefinition.Middleware(), 1)
 }
 
+func testMarshalForwardingTimeoutsToJSON(t *testing.T) {
+	definition := Definition{
+		ListenPath: "/*",
+		Upstreams: &Upstreams{
+			Balancing: "roundrobin",
+			Targets: Targets{
+				{Target: "http://test.com"},
+			},
+		},
+		ForwardingTimeouts: ForwardingTimeouts{
+			DialTimeout:           Duration(30 * time.Second),
+			ResponseHeaderTimeout: Duration(31 * time.Second),
+		},
+	}
+	jsonDefinition, err := json.Marshal(&definition)
+	require.NoError(t, err)
+	assert.Contains(t, string(jsonDefinition), "30s")
```

Comment:
add field name here and the line bellow as well to be alightly more specific, because we may have another duration fields in the definition, e.g. `"dial_timeout":"30s"`